docs/UserGuide.md

```diff
@@ -97,6 +112,43 @@ Format: `delete INDEX`
 Examples:
 * `list` followed by `delete 2` deletes the 2nd internship application in the list.
 
+### Finding an internship application: `find`
+
+Finds applications whose companies or positions contain any of the given keywords.
+
+Format: `find KEYWORD [MORE_KEYWORDS]`
+
+* The search is case-insensitive. e.g `google` will match `Google`
+* The order of the keywords does not matter. e.g. `Tech Micron` will match `Micron Tech`
+* Only the company and position is searched.
+* Only full words will be matched e.g. `ByteDance` will not match `ByteDances`
+* Applications matching at least one keyword will be returned (i.e. `OR` search). e.g. `Google Shopee` will return `Shopee HQ`, `Google Singapore`
+
+Examples:
+* `find Software` returns `Google` and `Wise` application whose position contain software keyword.
+![FindExample](images/FindExample.png)
```

Comment:
Perhaps the image can be updated to one containing the new displayed Date format?